File: Shuttle-Clone-User/app/src/main/java/com/ferri/userapp/model/DefaultResponse.java
package com.ferri.userapp.model;

import com.google.gson.annotations.SerializedName;

public class DefaultResponse{

	@SerializedName("message")
	private String message;

	@SerializedName("otp")
	private int otp;

	@SerializedName("status")
	private boolean status;

	public String getMessage(){
		return message;
	}

	public boolean isStatus(){
		return status;
	}

	public int getOtp(){
		return otp;
	}
}

File: Shuttle-Clone-Driver/app/src/main/java/com/ferri/driver/Model/DefaultResponse.kt
package com.ferri.driver.Model

import com.google.gson.annotations.SerializedName
class DefaultResponse {
    @SerializedName("message")
    val message: String? = null

    @SerializedName("otp")
    val otp: Int? = null

    @SerializedName("status")
    val isStatus = false

}